pom.xml

```diff
@@ -1499,7 +1499,21 @@
             <dependency>
                 <groupId>org.apache.kafka</groupId>
                 <artifactId>kafka-clients</artifactId>
-                <version>2.4.1</version>
+                <version>3.2.0</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.slf4j</groupId>
+                        <artifactId>slf4j-api</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.lz4</groupId>
+                        <artifactId>lz4-java</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>com.github.luben</groupId>
+                        <artifactId>zstd-jni</artifactId>
```

Comment:
These exclusions (especially lz4 & zstd) are not safety as far as I tested locally. Please add tests using `compression.type` option.

Reply:
@ebyhr 
Can you please mention how you've tested it locally so that I could reproduce it?

As far as I can see - non of the existing functionality is modified by this commit.

If we need to add tests for compression types - it should be a **separate** issue/bug / missing feature, but I don't think it should be bound to this PR

Reply:
@kobihikri This PR seems to break reading ZSTD/LZ4 compressed topics which was working earlier - so it's a regression. The fact that @ebyhr found this via manual testing is not relevant. Since we now know this PR breaks existing feature we should add tests and fix it.

We can add tests as a separate PR but this PR cannot be merged before it.

Reply:
@hashhar 

The manual testing is relevant as I want to reproduce the problem - so that I can verify that I have fixed it (in a separate PR) :-)